lib/empathy/emp_json/helpers/slices.rb

```diff
@@ -56,7 +56,8 @@ def field_from_record(record, field)
         end
 
         def retrieve_id(resource)
-          return resource.to_s if resource.is_a?(URI) || resource.is_a?(RDF::URI) || resource.is_a?(RDF::Node)
+          return RDF::URI(resource) if resource.is_a?(URI)
```

Comment:
This method was meant to return a string

https://github.com/empathy-tools/emp_json_rb/blob/main/test/empathy/emp_json/slices_test.rb#L18